_pages/algorithm-development/your-first-algo.md

```diff
@@ -13,216 +13,109 @@ redirect_from:
-
-## (Optional) Deleting your Algorithm
+## Next Steps
 
-If your algorithm has never been published, or has only been published for private use, you have the option to delete it when it is no longer needed. To do so, go to your algorithm's page, click on the **Settings** tab, and find the red **Delete** button at the bottom.
+Now that you've published your first simple algorithm on the platform, you're empowered to start developing much more complex and useful algorithms. You can write them from scratch, adopt and modify open-source algorithms, and even pipeline multiple algorithms together. Once published, your algorithms will become versioned API endpoints that you can consume from external applications with just a few lines of code!
 
-<img src="{{site.cdnurl}}{{site.baseurl}}/images/post_images/algo_dev_lang/delete_algorithm.png" alt="Delete algorithm" class="screenshot img-sm">
+### Language Guides
 
-If you have ever published your Algorithm with "Public" selected, it cannot be deleted. This is to ensure that others' code will not break if they have written algorithms or client code that depends on your algorithm.
+Now that you know the basics, you can check out the <a href="{{site.baseurl}}/algorithm-development/languages">Language Guides</a> to learn how to write algorithms in whichever language you prefer; we support R, Python, Rust, Ruby, Java, Scala, and JavaScript.
 
-## Wrapping Up
+### Connecting to Data
 
-Now that you've published your first simple algorithm on the platform, you are empowered to develop more complex and useful algorithms. You can write them from scratch, adopt and modify open-source algorithms, or even chain multiple pre-existing algorithms together to make a micro service that you can call with the Algorithmia API! 
+As you saw in this guide, you can create simple algorithms that don't read or write any data to disk. However, most algorithms you develop will need to be able to interact with some sort of data source.
 
-## Next Steps
+As an integration-first platform, we encourage you to connect to whatever data source(s) you're already using. We also enable you to host data directly on our platform for lower-latency access, and we even have several native integrationss with cloud-hosted data platforms.
 
-After you've finished this tutorial, you'll probably want to check out the <a href="{{site.baseurl}}/algorithm-development/languages">Language Guides</a> for how to write algorithms in the language you prefer, such as R, Python, Rust, Ruby, Java, Scala, or JavaScript.
+See our <a href="http://docs.algorithmia.com/#data">Data API</a> docs and our <a href="{{site.baseurl}}/data">Developer Center</a> docs to get started connecting to data sources. For a deeper dive tutorial, you can also check out our [Training Center](https://training.algorithmia.com/path/enterprise-20-2-data-scientist/enterprise-20-2-using-data-sources).
```

Comment:
This is an old url, can avoid a couple of directs by hitting the new url directly
```suggestion
See our <a href="https://algorithmia.com/developers/api/#data">Data API</a> docs and our <a href="{{site.baseurl}}/data">Developer Center</a> docs to get started connecting to data sources. For a deeper dive tutorial, you can also check out our [Training Center](https://training.algorithmia.com/path/enterprise-20-2-data-scientist/enterprise-20-2-using-data-sources).
```

Reply:
🙏🏼  thanks